package.json

```diff
@@ -155,45 +155,44 @@
     "app-root-path": "^2.2.1",
     "async": "^2.6.2",
     "audit-ci": "^3.1.1",
-    "autoprefixer": "^7.1.1",
+    "autoprefixer": "^9.0.0",
     "axe-webdriverjs": "^2.2.0",
     "babel-loader": "^8.0.0",
     "babel-plugin-auto-await": "^0.4.2",
-    "babel-upgrade": "0.0.23",
     "browserstack-local": "^1.4.5",
     "chai": "^4.2.0",
     "chalk": "^2.4.2",
-    "chromedriver": "^92.0.0",
+    "chromedriver": "^93.0.0",
```

Comment:
Why not `"chromedriver": "^94.0.0"`? That's what I'm using on my other branch

Reply:
Oh. That's odd. All the tests passed so didn't think to check that it was outdated. I'll bump it to the latest anyway.